CONP_DATS_fields.md

```diff
@@ -1,47 +1,53 @@
+|Primary Publications|primaryPublications|**RECOMMENDED**| The primary publication(s) associated with the dataset, usually describing how the dataset was produced.| The primary publication(s) associated with the dataset, usually describing how the dataset was produced.|
+|Title|Primary Publications -> Title||The name of the publication, usually one sentence or short description of the publication.|The name of the publication.|
+|Publication Venue|Primary Publications -> Publication Venue||The name of the publication venue where the document is published (if applicable).|The name of the publication venue where the document is published (if applicable).|
+|Authors|Primary Publications -> Authors||Authors of the publication.|Authors of the publication.|
+|Affiliations|Primary Publications -> Authors -> Affiliations||Author affiliations.|Authors affiliations.|
+|Date|Primary Publications -> Dates||Relevant dates for the publication. If you provide a date, it must come with a description of the date.|Relevant dates for the publication. If you provide a date, it must come with a description of the date (i.e.: first submission, final approval, date of publication, ...).|
+|Identifier|Primary Publications -> Identifier|| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Identifier|Primary Publications -> Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying an entity locally to a system or globally.|
+|Identifier Source|Primary Publications -> Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the publicationt. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Dimensions|dimensions|**RECOMMENDED**| The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|
+|Identifier|identifier|| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Identifier|Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying an entity locally to a system or globally.|
+|Identifier|Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one. |
```

Comment:
same comment about the identifier source as above

Reply:
And same comment as about "access"; if we have clearly defined description for each field I think the repetition in the description of a subheader line (like 41 here) is not useful.

Reply:
@cmadjar If using Zenodo, then the dataset should not have a DOI yet. After uploading, then the DOI generated by Zenodo could be added to the dats file (or the Zenodo process will add it IIRC)

@emmetaobrien same comment, I added every help comment on the GUI here so we have a single place as reference.

Reply:
see reply to https://github.com/CONP-PCNO/conp-documentation/pull/53/files#r519925395

Reply:
same changes done.

Reply:
```suggestion
|Identifier Source|Primary Publications -> Identifier -> Identifier Source|**RECOMMENDED**| Information about the organisation/namespace responsible for minting the identifier.| Information about the organisation/namespace responsible for minting the identifier. It must be provided if the identifier is provided.|
```

Reply:
Change made except for "Publication" as it is in the dataset section.

Reply:
right. ooopsie ;)